Lib/test/test_xxlimited.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+from test.support import import_helper
+import types
+
```

Comment:
nitpick: PEP 8 suggests two empy lines after imports, if I recall properly :-D

Reply:
The following lines also imports ;)